### ISSUE TITLE
Add ability to push destroy config

### ISSUE THREAD

**Issue by hatched-DavidMichon** [hashicorp/tfe-cli#27]
Currently it's not possible to destroy via the pushconfig command. Would be very helpful to have this feature.

**Comment by fprimex** [comment]
👋 Right now `tfe` doesn't support any kind of apply operations. A destroy would take three steps - set the appropriate environment variable to allow it, perform a destroy plan, apply the destroy plan.

The variable can be pushed and performing a destroy plan can be added as an option. Adding an apply for the resulting plan would be a new undertaking. I'm not opposed to venturing into apply territory, but I need to wait on that for at least a week.

**Comment by fprimex** [comment]
Have a look at:

https://github.com/hashicorp/tfe-cli/tree/push-destroy

This will get the destroy plan going, but there still is no way presently to apply a plan. I will look at doing that as well when I'm able. Let me know how this works for you.

**Comment by hatched-DavidMichon** [comment]
Hi @fprimex, thanks very much! Looks promising! Yes it works for me for sure. Whenever you have time to dedicate to this will fit me.

**Comment by fprimex** [comment]
merged into master
